testing/tests/DevExpress.ui.widgets.dataGrid/dataController.tests.js

```diff
@@ -3853,6 +3853,30 @@ QUnit.module('Virtual rendering', { beforeEach: setupVirtualRenderingModule, aft
         assert.strictEqual(this.dataController.pageCount(), 5, 'page count');
     });
 
+    // T750279
+    QUnit.test('setViewportItemIndex should be called for rowsScrollController and dataSource with same args after loadIfNeed call', function(assert) {
+        // act
+        this.dataController.setViewportPosition(50);
+
+        // assert
+        assert.strictEqual(this.dataController.getContentOffset('begin'), 50);
+
+        // act
+        const rowsScrollControllerSpy = sinon.spy(this.dataController._rowsScrollController, 'setViewportItemIndex');
+        const dataSourceSpy = sinon.spy(this.dataController._dataSource, 'setViewportItemIndex');
+
+        this.dataController.loadIfNeed();
+
+        // assert
+        const rowsScrollControllerCall = rowsScrollControllerSpy.getCall(0);
+        const dataSourceCall = dataSourceSpy.getCall(0);
+
+        assert.ok(rowsScrollControllerCall, 'rowsScrollController\'s setViewportItemIndex is called');
+        assert.ok(dataSourceCall, 'dataSource\'s setViewportItemIndex is called');
+
+        assert.equal(rowsScrollControllerCall.args[0], dataSourceCall.args[0], 'setViewportItemIndex call args are same');
```

Comment:
Check call count for this methods.
Also check parameter values instead of args equality.